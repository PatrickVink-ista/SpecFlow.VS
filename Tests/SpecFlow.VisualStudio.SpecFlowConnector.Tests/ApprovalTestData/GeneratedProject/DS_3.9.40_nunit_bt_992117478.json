{
    "GeneratorOptions": {
        "featureFileCount": 1,
        "scenarioPerFeatureFileCount": 1,
        "targetFramework": "net48",
        "SpecFlowPackageVersion": "3.9.40"
    }
}
