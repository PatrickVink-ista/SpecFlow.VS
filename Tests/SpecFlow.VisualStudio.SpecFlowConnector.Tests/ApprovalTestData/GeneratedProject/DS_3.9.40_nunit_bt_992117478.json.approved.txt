{
  "stepDefinitions": [
    {
      "type": "Then",
      "regex": "^tellus \u0022(.*)\u0022 nulla$",
      "method": "IdNecAtLigulaSteps.ThenErosAtCongueMollis(String)",
      "paramTypes": "s",
      "expression": "tellus \u0022(.*)\u0022 nulla",
      "sourceLocation": "#0|10|9|12|10"
    },
    {
      "type": "Given",
      "regex": "^Curabitur sem in$",
      "method": "IdNecAtLigulaSteps.GivenInMassaDiamDui()",
      "expression": "Curabitur sem in",
      "sourceLocation": "#0|16|9|18|10"
    },
    {
      "type": "When",
      "regex": "^nostra ullamcorper tortor in$",
      "method": "IdNecAtLigulaSteps.WhenAugueAugueLaoreetNulla()",
      "expression": "nostra ullamcorper tortor in",
      "sourceLocation": "#0|22|9|24|10"
    }
  ],
  "sourceFiles": {
    "0": "<<targetFolder>>\\StepDefinitions\\IdNecAtLigulaSteps.cs"
  },
  "typeNames": {}
}
